docs/recipes/recipe06.ipynb

```diff
@@ -195,7 +196,7 @@
     "policy2 = tc.Policy()  # parametric reform, reformC.json, eliminates EITC\n",
     "\n",
     "# TODO: Move to the web so this can be done standalone.\n",
-    "policy2.implement_reform(tc.Policy.read_json_reform('_static/reformC.json'))\n",
+    "policy2.implement_reform(tc.Policy.read_json_reform('github://PSLmodels:Tax-Calculator@master/docs/recipes/_static/reformC.json'))\n",
```

Comment:
To facilitate running on Google Colab, I changed the recipes to file reforms at URLs (need to use this URL format due to issue with `fsspec` dependency of paramtools having issues with other github URL formats at the moment, see [ParamTools Issue #128](https://github.com/PSLmodels/ParamTools/issues/128)).

This works fine, only issue would be if one wanted to change recipes and run locally before those are committed to the branches referenced.

Let me know if you think this is a problem or if you see other issues with this approach.

cc @MaxGhenis @MattHJensen